examples/angular-app/angular.json

```diff
@@ -26,7 +26,7 @@
             ],
             "styles": [
               "src/styles.css",
-              "src/assets/plugins/rest-import-bundle.css"
+              "src/assets/plugins/rest-import-ui.css"
```

Comment:
have we changed this in file in babel configs as well?

Reply:
Yes.. Now we have the file name as 'rest-import-bundle' in all places